indexer/analyzer_tasks.go

```diff
@@ -382,7 +409,21 @@ func (t *systemEventCreatorTask) getDelegationChangedSystemEvents(currSeqs, prev
 	return systemEvents, nil
 }
 
-func (t *systemEventCreatorTask) getValueChangeSystemEvents(currValidatorSeqs, prevValidatorSeqs []model.ValidatorSeq, syncable *model.Syncable) ([]model.SystemEvent, error) {
+func (t *eraSystemEventCreatorTask) getPrevValidatorEraSequences(payload *payload) ([]model.ValidatorEraSeq, error) {
+	var prevEraSequences []model.ValidatorEraSeq
+
+	if payload.CurrentHeight > t.cfg.FirstBlockHeight {
+		var err error
+		prevEraSequences, err = t.validatorEraSeqDb.FindByEra(payload.Syncable.Era - 1)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return prevEraSequences, nil
+}
```

Comment:
Another fyi. It seems it's possible to do this :) 
```suggestion
func (t *eraSystemEventCreatorTask) getPrevValidatorEraSequences(payload *payload) (prevEraSequences []model.ValidatorEraSeq, err error) {
	if payload.CurrentHeight > t.cfg.FirstBlockHeight {
		prevEraSequences, err = t.validatorEraSeqDb.FindByEra(payload.Syncable.Era - 1)
	}

	return prevEraSequences, err
}
```